README.md

```diff
@@ -39,7 +39,7 @@ var el = {
 ### Converting Javascript Values into Elements
 
 ```javascript
-var minim = require('minim');
+var minim = require('minim').init();
```

Comment:
> I also think every registry should just default to loading the elements included with Minim

The problem is, if you don't do what I've done here (or actually store the registry in each reference), you MUST always use the default registry. That is because as we currently have it, each instance only knows of one registry, and even if you create your own, those instances will only use the default. 

So the problem isn't really loading the including elements, it's that those elements (in master) only know of one registry.

If that makes sense.